File: app/src/main/scala/com/example/scala/MainActivity.kt
package com.example.scala

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.scala.databinding.MainBinding


class MainActivity : AppCompatActivity() {
    private lateinit var binding: MainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = MainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}

File: app/src/main/scala/com/example/scala/MainActivity.scala
package com.example.scala

import android.app.Activity
import android.os.Bundle
import android.widget.{Button, Toast}
//import com.example.scala.datadinding.MainBinding

class MainActivity extends Activity() {
    override def onCreate(savedInstanceState: Bundle): Unit = {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main)
        findViewById(R.id.btn).asInstanceOf[Button].setOnClickListener(_ -> {
            Toast.makeText(this, "Click", Toast.LENGTH_SHORT).show()
        })
//        var binding = MainBinding.inflate(getLayoutInflater)
//        setContentView(binding.root)
    }
}